app/views/shared/_product.html.erb

```diff
@@ -0,0 +1,58 @@
+<%# product card %>
+
+<% 
+# puts "==="
+# puts product.inspect
+# puts "==="
+# puts product.id
+# puts "==="
+
+%>
+
+<%# display card %>
+<% cache @product do %>
+  <div class="product-card Polaris-Layout__Section Polaris-Layout__Section--oneThird">
+    <div class="Polaris-Card">
+      <div class="Polaris-Card__Section">
+        <div class="product-card__image">
+          <%= image_tag(product.image_url) %>
+          <p>&nbsp;</p>
+          <p class="product-card__price"><%= number_to_currency(product.price, unit: "€") %></p>
+          <p>&nbsp;</p>
+          <h2 class="Polaris-Heading"><%= product.title %></h2>
+        </div>
+      </div>
+      <div class="Polaris-Card__Section">
+        <p class="product-card__description Polaris-TextStyle--variationSubdued"><%= formatted_description(product.description) %></p>
+      </div>
+
+      <%# are we showing the editing links? %>
+      <% if local_assigns[:manage] %>
```

Comment:
Had to look up what `local_assigns` does. I like it!

One thought on the subject though, if it's possible this _won't_ be rendered at all, would it be better extracted to a partial that gets called on if `manage` is passed when rendering `_product`?

Reply:
Cool yeah, so basically abstract the manage part to it's own partial. Will do that :)